macros/EmbedTest262ReportResultsTable.ejs

```diff
@@ -0,0 +1,12 @@
+<%
+// Embeds a Test262.report results table
+//
+// Parameters:
+//  $0 - A "feature" name from test262.report
+//
+//  Example calls:
+// {{EmbedTest262ReportResultsTable("globalThis")}}
+// {{EmbedTest262ReportResultsTable("BigInt")}}
+//
+%>
+<iframe frameborder="0" scrolling="no" style="width: 100%; height: 300px;" src="https://test262.report/embed/features/<%= $0 %>?engines=chakra%2Cjavascriptcore%2Cspidermonkey%2Cv8&summary=true&include-browsers=true"></iframe>
```

Comment:
Does there need to be vetting of that `$0` parameter? I.e. if someone edits a wiki page with:
```
{{EmbedTest262ReportResultsTable('"><script>alert()</script>')}}
```
Or is that already taken care of in the kuma-to-kumascript stuff? @escattone ?

Reply:
@peterbe HTML sanitization appears to be built in, ie. `'"><script>alert()</script>'` would become: `&#34;&gt;&lt;script&gt;alert()&lt;/script&gt;`